### PR TITLE
[VariationBundle] CS fixes

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q                    | A
| -------------       | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes
| Deprecations? | no
| Fixed tickets | no
| License       | MIT

### CODE REVIEW

File: src/Sylius/Bundle/VariationBundle/Validator/Constraint/VariantUnique.php [hunk 1]
```diff
@@ -22,7 +22,14 @@
  */
 class VariantUnique extends Constraint
 {
+    /**
+     * @var string
+     */
     public $message = 'This property must be unique';
+
+    /**
+     * @var
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/VariationBundle/Validator/Constraint/VariantUnique.php]: `@var mixed`?

**Comment by aRn0D** [comment]
:+1:

**Comment by stloyd** [comment]
Such commits should contain `[ci skip]` note to avoid wasting resources on Travis-CI & blocking PRs that actually test something.

**Comment by michalmarcinkowski** [comment]
:+1:

**Pull request merged**